contracts/GeVault.sol

```diff
@@ -290,7 +293,7 @@ contract GeVault is ERC20, Ownable, ReentrancyGuard {
       liquidity = tSupply * valueX8 / vaultValueX8;
     }
     
-    rebalance();
+    deployAssets();
```

Comment:
@Keref don't we miss an if(isActive) here?

Reply:
It's already at the beginning of the function